README.md

```diff
@@ -93,6 +93,17 @@ The `fiori-fundamentals` library follows [Semantic Versioning](https://semver.or
 
 Merges to the `master` branch will be published as a prerelease. Prereleases will include an **rc** version (_e.g._ `[MAJOR].[MINOR].[PATCH]-rc.[RC]`).
 
+The following circumstances will be considered a MAJOR or BREAKING change:
+* Droppping existing classnames, css variables, color names, color groups, spacing parameters
+* The existing underlying HTML markup of a component is altered
+* An existing unit test is altered to account for either of the above
+
+The following circumstances will NOT be considered a MAJOR or BREAKING change:
+* Introducing new  classnames, css variables, color names, color groups, spacing parameters
+* Non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
+* An existing unit test is altered to account for non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
```

Comment:
(nit) I'd like this bullet point to be the final one.

Reply:
done